### PR TITLE
Stasis beds now are surgery-friendly

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
## About The Pull Request

Adds a modifier to probability for SBs, making them better than both tables and beds but not as good as dedicated surgery tables.

## Why It's Good For The Game
It allows for a decision making process that previously would rarely exist since a 0% bonus compared to a 100% bonus is pretty steep (if you knew it didn't have one). Now you trade 10% for the perks of being in stasis.

## Changelog
:cl: CobbyzzzZzzz
balance: Stasis are now the second best option for surgery probability (surgical tables are still number 1)
/:cl:

### CODE REVIEW

File: code/modules/surgery/surgery.dm [hunk 1]
```diff
@@ -113,6 +113,8 @@
 
 	if(locate(/obj/structure/table/optable, T))
 		propability = 1
+	else if(locate(/obj/machinery/sleeper, T))
```
Comment (YPOQ) [inline code/modules/surgery/surgery.dm]: stasis beds are https://github.com/tgstation/tgstation/blob/8ad3beaab6300f25b00406f829fa2c4ef96ece63/code/game/machinery/stasis.dm#L2

Comment (ExcessiveUseOfCobblestone) [reply code/modules/surgery/surgery.dm]:
```suggestion
	else if(locate(/obj/machinery/stasis, T))
```
lol

**Comment by JJRcop** [comment]
image

**Comment by ExcessiveUseOfCobblestone** [comment]
the government put a word filter in my loyalty implant

**Comment by fludd12** [comment]
This is the first time I looked at the code for how surgery probability is calculated and... _wow_ is this in need of a refactor. Not that surgery as a whole doesn't already.

who even let `propability` into the codebase

**Comment by ExcessiveUseOfCobblestone** [comment]
featore

**Review by SpaceManiac — APPROVED** [review]
pro:b:ability

**Pull request merged**